doc/api_ref/pubkey.rst

```diff
@@ -120,14 +120,27 @@ Kyber
 
```

Comment:
[Arbitrary code location]
pubkey.rst:1167
```rst
Botan implements the following KEM schemes:

1. RSA
#. Kyber
#. FrodoKEM
#. McEliece
```
We can now write something like `#. ML-KEM (Kyber)`

Also pubkey.rst:1176 
```rst
Code Example: Kyber
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The code below demonstrates key encapsulation using the Kyber post-quantum scheme.

.. literalinclude:: /../src/examples/kyber.cpp
   :language: cpp
```
... can be adopted accordingly. Maybe we should change the Kyber example to an ML-KEM example and rename the file accordingly.

Reply:
Done. Also renamed the example to `ml_kem.cpp`.